src/app/system/manage-data-tables/manage-data-tables.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+    Create Data Table
+  </button>
+</div>
+
+<div class="container">
+
+  <div fxLayout="row" fxLayoutGap="20px">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="dataTableName">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Data Table Name </th>
+        <td mat-cell *matCellDef="let dataTable"> {{ dataTable.applicationTableName }} </td>
```

Comment:
You have swapped the properties. This should be registeredTableName.
See here: https://github.com/openMF/web-app/issues/170